weather/src/com/weather/replenished/Replenished.java

```diff
@@ -0,0 +1,4 @@
+package com.weather.replenished;
+
+public class Replenished {
```

Comment:
Is this a noun?

Reply:
It was a bad name. I meant "Precipitation"